ext/ftp/tests/ftp_ssl_connect_error.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Test ftp_ssl_connect() function : error conditions
+--SKIPIF--
+<?php
+$ssl = 1;
+require 'skipif.inc';
+if (!function_exists("ftp_ssl_connect")) die("skip ftp_ssl is disabled");
+?>
+--FILE--
+<?php
+echo "*** Testing ftp_ssl_connect() function : error conditions ***\n";
+echo "\n-- Testing ftp_ssl_connect() function on failure --\n";
+var_dump(ftp_ssl_connect('127.0.0.1'));
```

Comment:
No guarantee there isn't a server listening on localhost.  Maybe try a hostname of `does-not-exist.invalid`